src/Video/RenderProgram/StaticRenderProgram.hpp

```diff
@@ -18,6 +18,12 @@ namespace Video {
             
             /// Destructor.
             ~StaticRenderProgram();
+            /// Bind shadow render program.
+            /**
+            * @param viewMatrix The camera's view matrix.
```

Comment:
Incorrectly aligned doxygen comment. This isn't the camera's view matrix; it's the light's view matrix.